packages/manager/modules/hub/src/components/support/ticket.class.js

```diff
@@ -0,0 +1,24 @@
+import capitalize from 'lodash/capitalize';
+
+export default class SupportTicket {
+  constructor(ticket) {
+    Object.assign(this, ticket);
+  }
+
+  getDisplayName() {
+    return capitalize(this.serviceName);
+  }
+
+  getStateCategory() {
+    switch (this.state) {
+      case 'open':
+        return 'success';
+      case 'closed':
+        return 'info';
+      case 'unknown':
+        return 'warning';
+      default:
+        return 'error';
```

Comment:
Maybe we should reuse https://github.com/ovh/manager/blob/master/packages/manager/modules/support/src/tickets/ticket/ticket.class.js#L3 and add what its needed

Reply:
yep please see : https://github.com/ovh/manager/pull/2363